doc/related_projects.rst

```diff
@@ -150,6 +159,17 @@ and tasks.
 - `multiisotonic <https://github.com/alexfields/multiisotonic>`_ Isotonic
   regression on multidimensional features.
 
+- `libOPF <https://github.com/LibOPF/LibOPF>`_ Optimal path forest classifier
+
+- `pyensemble <https://github.com/dclambert/pyensemble>`_ An implementation of
```

Comment:
I felt this was a trade-off between the two but maybe Auto-ML might be more natural for it. I'll make the change.